src/components/RulesSelector.jsx

```diff
@@ -27,19 +29,14 @@ const makeListItems = (filters, onRuleSelected) => (
 const RulesSelector = ({ filters, onRuleSelected }) => {
   const listItems = makeListItems(filters, onRuleSelected);
   return (
-    <div>
+    <div className="rule-selector-container">
```

Comment:
There are certain CSS selectors that don't directly translate to JavaScript object. So if its going to be complicated to use objects for certain selectors, then I'm not against keeping them. But for the sake of consistency, it would be good to try to use objects when possible.